packages/sdk/test/unit/Mapping.test.ts

```diff
@@ -97,25 +97,35 @@ describe('Mapping', () => {
     })
 
     it('max size', async () => {
-        const SIZE = 3
+        const maxSize = 3
```

Comment:
Good rename. Let's use naming style of constants.

```suggestion
        const MAX_SIZE = 3
```